docs/developing/information/indexers.md

```diff
@@ -1,429 +1,186 @@
-* **API** is the acronym for _Application Programming Interface_, which is a software intermediary that allows two applications to talk to each other.
+There are two main types of blockchain indexers: full and selective.
 
-### Focus on BlockWatch Indexer (TzIndex)
+### Full indexers
 
-The Blockwatch Indexer [TzIndex](https://github.com/blockwatch-cc/tzindex) is used for the [TzStats explorer](https://tzstats.com/).
+Full indexers process and write all data from blocks, from simple transactions to validator's node software versions.
+Blockchain explorers commonly use them to provide users with advanced blockchain analytics and allow them to search for any type of on-chain data.
+Also, those who host full indexers can offer public APIs that other projects can use without hosting the indexer themselves.
 
-The **Blockwatch indexer** replaces the slow and expensive SQL datastore with a high-performance columnar database that allows for extremely fast analytical queries.
+You can get data from these full indexers, which allow you to find almost any information in the Tezos blockchain:
 
- **Columnar database** is a column-oriented storage for databases. It is optimized for fast retrieval of data columns, for example for analytical applications. It significantly reduces the overall disk I/O requirements and limits the amount of data you need to load from the disk.
+- [TzKT](https://api.tzkt.io/)
+- [TzPro](https://docs.tzpro.io/)
+- [TzIndex](https://github.com/blockwatch-cc/tzindex)
 
-It's a custom-made database for blockchain analytics. Avoiding the storage bottleneck allows for more complex data processing.
+### Selective indexers
 
-> **Storage bottleneck** is a situation where the flow of data gets impaired or stopped completely due to bad performance or lack of resources.
+Selective indexers store only selected data, which means that they need less space and resources to maintain.
+Creating a selective indexer requires you to write the logic for the indexes, including what data to fetch, how to process it, and how to store it.
 
-State updates happen at each block, which means all the balance updates are always verified, and the indexer will follow chain reorganizations in real-time.
+Usually, they are used in projects that require only specific on-chain data, such as active user balances, balances of their smart contracts, and NFT metadata.
+You can optimize a custom selective indexer for fast execution of specific project queries.
+You can also use them to work with data in the way that you want to, such as storing token amounts with decimal places.
 
+You can use these frameworks to create your own selective indexer to provide the data that you need in the format that you need:
 
-## A simple explanation of how indexers work
+- [Que Pasa](https://github.com/tzConnectBerlin/que-pasa)
```

Comment:
The last commit for Que Pasa 2 year ago. Perhaps it's better to check if it works before.

Reply:
Ok, I'm going through these instructions on opentezos to see if I can get que pasa running:
https://opentezos.com/dapp/indexers/que_pasa/

Reply:
I got que-pasa to work with some hacks:

- It doesn't like that some RPCs return 404s including the data, so I used https://rpc.tzbeta.net per your suggestion
- `--enable-bcd` doesn't seem to work (it throws timeout errors), so I didn't use it
- I had to manually go into config.rs and set NODE_URL and DATABASE_URL because it doesn't seem to respect environment variables

But after all that I can run it and see it populate the index with data about the contract.